build/pyodide_bucket.bzl

```diff
@@ -0,0 +1,3 @@
+PYODIDE_PACKAGE_BUCKET_URL = "https://pub-45d734c4145d4285b343833ee450ef38.r2.dev/20240305/"
```

Comment:
Maybe comment here that it's factored out b/c it's shared between WORKSPACE and src/pyodide/BUILD.bazel